django/README.md

```diff
@@ -15,4 +17,26 @@ python manage.py migrate
 # Log into database to inspect the new table
 docker exec -t -i cthub_db_1 psql -U postgres
 # run this: \d <new_table_name>
+```
+
+If you need to change a migration you can back up by doing the following:
+1. List the migrations to find where you want to go back to
+> `python manage.py showmigrations`
+2. Move migration to the appropriat state
+> `python manage.py migrate --fake api 0003_name_of_migration_before_one_to_redo`
+3. Delete the migration file
+4. Delete table if necessary
+5. Re-run migration
+> `python manage.py makemigrations`
+
+
+## Data Loads
+Copy the spreadsheet into the _api_ docker container.
+```bash
+docker cp 'EV_Fast-Charging Stations_20210520.xlsx' cthub_api_1:/tmp/
```

Comment:
You should be able to get away with just putting the file temporarily in django folder. Is the idea here so that we have a temporary folder somewhere where we just put our files for importing?

Reply:
That's a good idea.
The `/tmp` folder is always world writable in Linux systems. When copying stuff to a container I usually use that location.